Makefile

```diff
@@ -9,17 +9,17 @@ build/CMakeCache.txt:
 	$(MAKE) rebuild-cmake
 
 install-pkgs: rebuild-cmake
-	(cd build && ninja install-pkgs)
+	cmake --build build --target install-pkgs
 
 test: install-pkgs rebuild-cmake
-	(cd build && ninja check)
+	cmake --build build --target check
 
 # We rebuild the cmake file all the time.
 # We use "glob" in the cmakefile, and wouldn't otherwise notice if a new
 # file (for example a test) was added or removed.
 # It takes <1s on Linux to run cmake, so it doesn't hurt to run it frequently.
 rebuild-cmake:
 	mkdir -p build
-	(cd build && cmake .. -G Ninja)
+	cmake -B build -DCMAKE_BUILD_TYPE=Debug
```

Comment:
Why do we need to use Debug here?

Reply:
The non-ninja builder requires a build-type. Otherwise the tests don't work on Windows.